CMakeLists.txt

```diff
@@ -587,6 +588,91 @@ if (BUILD_TESTING)
             set_property(TEST ${test_target} PROPERTY TIMEOUT 7200)
         endforeach()
     endif()
+
+    if(S2N_FUZZ_TEST)
+        set(SCRIPT_PATH "${CMAKE_CURRENT_SOURCE_DIR}/tests/fuzz/runFuzzTest.sh")
+        file(GLOB FUZZ_TEST_SRCS "${CMAKE_CURRENT_SOURCE_DIR}/tests/fuzz/*.c")
+
+        file(GLOB TESTLIB_SRC "tests/testlib/*.c")
+        file(GLOB TESTLIB_HEADERS "tests/testlib/*.h" "tests/s2n_test.h")
+
+        add_library(fuzztest SHARED ${TESTLIB_HEADERS} ${TESTLIB_SRC})
```

Comment:
nit: we should probably be dealing with this in the same place that the unit tests do.

https://github.com/aws/s2n-tls/blob/9738cc7779e1426ba7379bde01472caec2e29cb2/CMakeLists.txt#L446-L450

But also it's probably fine to handle this as part of the next PR. I think once we switch to CMake idioms, this will generally tidy itself up and be much easier to work with.